.eleventy.js

```diff
@@ -87,6 +88,48 @@ module.exports = function (eleventyConfig) {
     `}
   );
 
+  eleventyConfig.addPairedShortcode('docLinks', (children, locale, stage, figma, github) => {
+    let stageChip = "";
+    let figmaLink = "";
+    let githubLink = "";
+    const langStrings = {
+      "en": {
+        "stage": "stage",
+        "figma": "Figma",
+        "github": "GitHub"
+      },
+      "fr": {
+        "stage": "phase",
+        "figma": "Figma",
+        "github": "GitHub"
+      }
+    }
+    if (stage) {
+      stageChip = `<div class="stage-chip"><span>${langStrings[locale].stage}</span><span>${stage}</span></div>`;
+    }
+    if (figma) {
+      figmaLink = `
+        <li class="figma-link">
+          <a href="${figma}" lang="en">${langStrings[locale].figma}</a>
+        </li>`;
+    }
+    if (github) {
+      githubLink = github;
+    } else {
+      githubLink = "https://github.com/cds-snc/gcds-components";
+    }
+    return `
+      <div class="doc-links">
```

Comment:
IF we move the stage chip into the list, we can remove the <div> and add the class to the <ul> instead.